packages/coordinator/tests/services/translation/response/release/release-response.spec.ts

```diff
@@ -314,7 +314,7 @@ describe("practitioner details", () => {
     const parentPrescription = getExampleParentPrescription()
     const prescription = parentPrescription.pertinentInformation1.pertinentPrescription
     prescription.author.AgentPerson.id._attributes.extension = "AuthorRoleProfileId"
-    prescription.author.AgentPerson.code._attributes.code = "AuthorJobRoleCode"
+    //prescription.author.AgentPerson.code._attributes.code = "AuthorJobRoleCode"
```

Comment:
You'll want to un-comment this again so that we can change the test on line 337 back

Reply:
done